picard/script/serializer.py

```diff
@@ -57,16 +57,28 @@ class PicardScriptType(IntEnum):
     FILENAMING = 2
 
 
-class ScriptImportExportError(Exception):
+class PicardScriptError(Exception):
```

Comment:
Do we really want to prefix all class names with "Picard" in this file? Maybe if there is confusion with other script related classes  it would be better to include the purpose about the classes in this file. Those are all classes to assist with (de)serializing the script.

E.g. instead of `PicardScript` use `ScriptSerializer`. Or if it is more about holding the info / metadata of the script call it `ScriptInfo` (I would avoid `ScriptMetadata` to avoid confusion with other metadata).

The naming here is a bit difficult, but I don't see adding the "Picard" prefix is improving the situation. It doesn't make it clear that these classes hold the script + metadata and allow writing it to yaml files.

Reply:
It was more a first step to ease renaming. I'm not happy with current names either.

So if we renamed PicardScript to ScriptSerializer (and related exceptions), then PicardTaggingScript to TaggingScriptInfo (and we have TaggingScriptSetting for now), do you think it could work?

Reply:
Yes, I think that would work. It reads much better IMHO.

Reply:
Done in latest commits